sdk/identity/azure-identity/azure/identity/aio/_credentials/auth_file.py

```diff
@@ -0,0 +1,69 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+import json
+import asyncio
+
+from .client_credential import ClientSecretCredential
+from azure.core.exceptions import ClientAuthenticationError
+
+
+def _parse_credentials_file(file_path):
+    with open(file_path, 'r') as f:
+        auth_str = f.read()
```

Comment:
Is there any way to do non-blocking read? Does it have to be non-blocking?

Reply:
@chlowell - Can you help with this?  Thanks

Reply:
To my knowledge, the standard library doesn't offer async file I/O. I'm fine with a blocking read.